src/components/Personalization/helper/prependHtml.js

```diff
@@ -41,7 +41,11 @@ export default (container, html) => {
     const element = elements[i];
     const firstChild = getFirstChild(container);
```

Comment:
Slightly pointless but would this work: 
`const firstChild = getFirstChild(container) || container;`

Reply:
nope, if the container does not contain any children then we need to `append` otherwise use `insertBefore`.